examples/streamed/misc/readers/reader-NYPL/webpub-viewer.js

```diff
@@ -637,26 +638,55 @@ define("ColumnsPaginatedBookView", ["require", "exports", "HTMLUtilities", "Brow
             this.hasFixedScrollWidth = (body.scrollWidth === originalScrollWidth);
             body.style.left = originalLeft + "px";
         };
+        // Get available width for iframe container to sit within
```

Comment:
This seems to work well on larger screens, but leaves no margin on mobile devices. I know that is a low priority for the moment, but want to make a note of it (if one hasn't already been made)